README.md

```diff
@@ -89,3 +89,7 @@ the differentiation. So we consider all the objects in the bucket irrespective o
 
 If a pipeline restarts during the snapshot, then the connector will start scanning the objects from the beginning of
 the bucket, which could result in duplications.
+
+### Useful resources
```

Comment:
do we have any resources? it's an empty header

Reply:
yes right, removed it